aafd/codec2_hal_default.te

```diff
@@ -0,0 +1 @@
+get_prop(codec2_hal_default, vendor_video_codec_prop)
```

Comment:
vendor_video_codec_prop and codec2_hal_default are declared in different directories(mixin groups), if one of them is disabled, there would be build errors for this. please do this:
1. add "BOARD_SEPOLICY_M4DEFS += module_aafd=true" to device/intel/mixins/groups/aaf/true/BoardConfig.mk
2. remove this line, and add   module_only(`aafd', `
  get_prop(codec2_hal_default, vendor_video_codec_prop)
')  to codec2/msdk-codec2/codec2_hal_default.te, just like wlan/load_iwlwifi/load_iwlwifi.te.